mc/Voyage-Mongo-Core.package/VOMongoRepository.class/instance/insertConcern.for..st

```diff
@@ -0,0 +1,3 @@
+persistence
+insertConcern: aConcern for: aClass
+	resolver insertConcern: aConcern for: aClass
```

Comment:
This might need to traverse the aClass and superclasses until isVoyageRoot is found? Any way to make this quick?

Reply:
btw. traverse the superclass is not needed. Just use #persistentClass it does what you want. I don't know what the insertConcern does because I don't know anything about mongo concerns. But is this really a configuration or something static. In the latter case it should be a class side method instead of this insertXXX